pyEDAA/Launcher/__init__.py

```diff
@@ -41,12 +41,16 @@
 from pathlib import Path
 from textwrap import dedent
 import time
+from typing import NoReturn
+
+from pyTooling.Decorators import export
 
 sub_path_bat  = Path("bin/vivado.bat")
 sub_path_vvgl = Path("bin/unwrapped/win64.o/vvgl.exe")
 match_line = "<!-- Product Version: Vivado v"
```

Comment:
I would like to change this to a RegExp.

Advantages:
* more robust pattern matching
* split version number into multiple parts  
  &rarr; reduce the following string operations

Reply:
```py
versionLineRegExp = re_compile(r"^<!--\s*Product\sVersion:\s+Vivado\s+v(?P<major>\d+).(?P<minor>\d+)(?:.(?P<patch>\d+))?\s+\(64-bit\)\s+-->")
```

Tested with [RegEx101.com](https://regex101.com/):  
![image](https://user-images.githubusercontent.com/956109/155988217-d4dd65ba-bcfc-40a7-97f4-661df246e6e4.png)